app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -0,0 +1,28 @@
+class IrsAttemptsEventsBatchJob < ApplicationJob
+  queue_as :default
+
+  def perform(subject_timestamp = Time.zone.now - 1.hour, dir_path = './attempts_api_output')
```

Comment:
another idea: let's make this write to the `/tmp` directory?

We can use ruby's `Tempfile` class to help make a file in there

Reply:
I'd suggested that we wanted to avoid Tempfile because it's eager to delete the file we just wrote. This will probably work fine once we're writing to S3, but for now it was just an annoyance and I figured we should treat what we're writing as if it persists.